sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPProfileDefinition.java

```diff
@@ -39,26 +40,27 @@
  * We currently also define two other profiles, see {@link DrupalProfile} and {@link PSR2Profile}.
  *
  */
-public final class PHPProfile extends ProfileDefinition {
+public final class PHPProfileDefinition extends ProfileDefinition {
+
+  public static final String SONAR_WAY_PROFILE = "Sonar way";
 
   private final RuleFinder ruleFinder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "RuleFinder"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)